imports/plugins/core/ui/client/components/button/buttonSelect.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from "react";
```

Comment:
Rename the file from `.jsx` to `.js`. The jsx extension is not necessary.

**Yes `button.jsx` will be renamed to `button.js` someday, just not today :)**